x-pack/plugins/security_solution/public/flyout/document_details/right/components/header_actions.tsx

```diff
@@ -36,8 +42,22 @@ export const HeaderActions: VFC = memo(() => {
     return `${value}&${FLYOUT_URL_PARAM}=${query.get(FLYOUT_URL_PARAM)}`;
   };
 
+  const { showAssistant, promptContextId } = useAssistant({
+    dataFormattedForFieldBrowser,
+    isAlert,
+  });
+
   return (
-    <EuiFlexGroup direction="row" justifyContent="flexEnd">
+    <EuiFlexGroup direction="row" justifyContent="flexEnd" alignItems="center" gutterSize="none">
```

Comment:
let's add `responsive={false}` here to avoid having the UI messed up on small screens
![Screenshot 2023-11-15 at 9 42 37 AM](https://github.com/elastic/kibana/assets/17276605/a3856d0e-6431-4cea-8a7e-9043359a7181)